src/app/therapist/manage-clients/manage-clients-list/manage-clients-list.component.html

```diff
@@ -64,7 +64,7 @@ <h4 class="modal-title">New Client</h4>
            ngModel
            required
            type="email"
-           pattern="^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$"
+           pattern="^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,63})+$"
```

Comment:
Can't believe I'm writing this.. But a bit of documentation would be great here. Such as what are we validating against here? Could be in style of:
`// Checking for existance of @, min 2 characters, max 63 characters`
And so on...